specification/transport_layer/can.tex

```diff
@@ -368,9 +366,9 @@ \subsubsection{Multi-frame transfers}
 In the latter case, the last CAN frame will contain only the metadata,
 as specified below in this section.
 
-Each CAN frame of a multi-frame transfer except the last one
+Each CAN frame of a multi-frame transfer (except the last one)
```

Comment:
This is neither a minor addition nor an expansion but a crucial part of the description. Do we really want it parenthesized?